hazelcast-sql-core/src/main/java/com/hazelcast/sql/impl/calcite/opt/physical/MapIndexScanPhysicalRel.java

```diff
@@ -103,9 +111,9 @@ public void visit(PhysicalRelVisitor visitor) {
     @Override
     public RelWriter explainTerms(RelWriter pw) {
         return super.explainTerms(pw)
-           .item("index", index.getName())
-           .item("indexExp", indexExp)
-           .item("remainderExp", remainderExp);
+            .item("index", index.getName())
```

Comment:
Descending flag (or flags) should be added to `explainTterms`. Otherwise, Calcite will treat nodes with different flag values as equivalent nodes, and it is hard to predict the consequences of it. Also, the signature will be used for `EXPLAIN` in the future, and flag value would be useful there as well.

Reply:
Addressed in https://github.com/hazelcast/hazelcast/pull/18013/commits/c364ce930090559c002db51c124c8f048a22192c